src/BaseStrategy.sol

```diff
@@ -5,6 +5,8 @@ import {IERC20Metadata as IERC20, Math, SafeERC20} from "src/Common.sol";
 import {BaseVault} from "src/BaseVault.sol";
 import {IBaseStrategy} from "src/interface/IBaseStrategy.sol";
 
+import {StrategyLib} from "src/library/StrategyLib.sol";
```

Comment:
@danoctavian I added the StrategyLib in a separate commit. 

imo it is not adding too much value, it doesn't change the storage by too much. 

WithdrawerStrategy with StrategyLib: 18,350 (6,226)
WithdrawerStrategy without StrategyLib: 18,432 (6,144)

So please let me know if you think I should revert this commit ?

Reply:
yeah agreed let's not do it then it seems extra for now